sample_data/cm360_conversions_sample_data.csvh

```diff
@@ -0,0 +1,2 @@
+floodlightConfigurationId,floodlightActivityId,encryptedUserId,mobileDeviceId,timestamp_micros,value,quantity,ordinal,limitAdTracking,childDirectedTreatment,gclid,nonPersonalizedAd,treatmentForUnderage,matchId,dclid,impressionId,userIdentifiers,conversion_kind
+13641571,16352759,"test","test","",1695955940000000,"","","1","","","0000000000","","","","","","",""
```

Comment:
Are all ids here mocked? I think it's best to remove any "real" data, even if not PII.

Reply:
Yes they are... they are from my own test campaign in dollar-a-day